toloka_provider/sensors/toloka.py

```diff
@@ -17,20 +17,24 @@ class WaitPoolSensor(BaseSensorOperator):
 
     :param pool: Either a `Pool` object or it's config or a pool_id value.
     :param toloka_conn_id: Airflow connection with toloka credentials.
+    :param success_on_reasons: Container of Pool.CloseReason
     """
 
-    template_fields: Sequence[str] = ('toloka_pool', 'toloka_conn_id')
+    template_fields: Sequence[str] = ('toloka_pool', 'toloka_conn_id', 'success_on_reasons')
 
     def __init__(
         self,
         *,
         toloka_pool: Union[Pool, Dict, str],
         toloka_conn_id: str = 'toloka_default',
+        success_on_reasons: Optional[Iterable[Union[Pool.CloseReason, str]]] = None,
         **kwargs,
     ) -> None:
         super().__init__(**kwargs)
         self.toloka_pool = toloka_pool
         self.toloka_conn_id = toloka_conn_id
+        self.success_on_reasons = [Pool.CloseReason(
+            reason) for reason in success_on_reasons] if success_on_reasons else list(Pool.CloseReason)
```

Comment:
```suggestion
            reason) for reason in success_on_reasons] if success_on_reasons  is None else list(Pool.CloseReason)
```
Currently code leads to unexpected behavior In case of  success_on_reasons = []

Reply:
Thanks, fixed